internal/logic/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/streemtech/panicbot/internal/logic
```

Comment:
Oh. You made them seperate modules.  This is excessive in this case.  You rarely need to use a go mod in another go mod.